src/components/game/components/board/components/buttons/Buttons.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const Buttons = ({id, onClick, visible}) => (
+  visible ? 
+  <button className="button" onClick={onClick}>
+    <div>{id}</div>
+  </button> : (<></>)
+);
```

Comment:
```suggestion
const Buttons = ({id, onClick}) => (
  <button className="button" onClick={onClick}>
    <div>{id}</div>
  </button>
);
```
The concern of the function is to create a button. Else it is its parent's decision not to do so. This align with the previous suggestion change.

Also it creates a ```Button``` so better make it singular instead of ```Buttons```

Reply:
Done.